schematic/manifest/generator.py

```diff
@@ -1633,11 +1634,14 @@ def create_manifests(
         title: Optional[str] = None,
         strict: Optional[bool] = True,
         use_annotations: Optional[bool] = False,
```

Comment:
Should path_to_data_model now be optional as well?

Reply:
My understanding is that we still need the jsonld data model for other operations